app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import shutil
+
+
+def copy_file(command: str) -> str:
+    file_1, file_2 = command.split()[1], command.split()[2]
```

Comment:
Okay if we will have 100 arguments solution will be the same?
1. https://github.com/avkpol/py-copy-file/blob/develop/checklist.md#code-efficiency
2. Validate that command have 3 arguments
3. Use correct variables names

Reply:
1. I stuck with the task's readme, cp command accepts only two arguments, doesn't it? Not 100.  
2. Should I  validate that the command has the "cp" keyword of the command itself and two more filename arguments?